spec/bolt/config_spec.rb

```diff
@@ -240,6 +240,15 @@
       }
       expect { Bolt::Config.new(boltdir, config) }.not_to raise_error
     end
+
+    it "warns when extra keys are present in 'apply_settings'" do
```

Comment:
This test is failing since we're not warning on invalid keys.

Reply:
Oh ha, I made this change locally and it looks like it got caught by my local gitignore. Thanks!